src/contract/table.ts

```diff
@@ -142,38 +143,40 @@ export class Table<RowType = any> {
      *  Each key-value pair in the queryParams object corresponds to a field and its expected value in the table.
      * @returns {Promise<TableRow>} Promise resolving to a single table row.
      */
-    async get(
-        queryValue?: API.v1.TableIndexType | string,
-        {scope = this.account, index, key_type}: QueryOptions = {}
-    ): Promise<RowType> {
+    async get(value?: API.v1.TableIndexType | string, params: QueryParams = {}): Promise<RowType> {
         const fieldToIndexMapping = this.getFieldToIndex()
 
         const tableRowsParams = {
             table: this.name,
             code: this.account,
-            scope,
+            scope: params.scope || this.account,
             type: this.rowType!,
             limit: 1,
-            lower_bound: wrapIndexValue(queryValue),
-            upper_bound: wrapIndexValue(queryValue),
-            index_position: index ? fieldToIndexMapping[index].index_position : 'primary',
-            key_type: key_type,
+            lower_bound: wrapIndexValue(value),
+            upper_bound: wrapIndexValue(value),
+            index_position: params.index
+                ? fieldToIndexMapping[params.index].index_position
+                : 'primary',
+            key_type: params.key_type,
             json: false,
         }
 
-        let {rows} = await this.client!.v1.chain.get_table_rows(tableRowsParams)
+        const {rows} = await this.client!.v1.chain.get_table_rows(tableRowsParams)
+        let [row] = rows
 
         if (!this.rowType) {
-            rows = [
-                Serializer.decode({
-                    data: rows[0],
-                    abi: this.abi,
-                    type: this.tableABI.type,
-                }),
-            ]
+            row = Serializer.decode({
+                data: row,
+                abi: this.abi,
+                type: this.tableABI.type,
+            })
+        }
+
+        if (params.json) {
```

Comment:
Hmm so it won't objectify data if json isn't true? Should we make that the default? 🤔

Reply:
Correct, the data will come back using core name types. 

This isn't the `json` parameter passed into `get_table_rows`, this is kinda a new parameter that is either:

- `json: false` (the default) the call will return core typed data
- `json: true` converts all the core typed data to a plain JSON object before returning it to the app.

Reply:
Maybe we should rename this, to avoid confusion with the parameter you'd use for `get_table_rows`, even though it kinda does the same thing 🤔 

By default, every call to `get_table_rows` I think will have `json: false`.

Reply:
Ahh ok that makes sense, I got confused because `Serializer.objectify(row)` doesn't do what I thought it did 😅 I think that the `json` param is fine here, but I guess that one other option is to have a `serialize` option that can take the "json" value (and potentially other values in the future??).

Reply:
I'm not sure what that'd do based on your comment, but it's definitely something we can explore! 👍 

And yeah - the `Serializer.objectify` I think literally just calls `JSON.parse(JSON.stringify())` on something to convert all the typed data (Name, Asset, etc) and makes it plain ol' JSON with strings and numbers.